vendor.mod

```diff
@@ -7,13 +7,13 @@ module github.com/docker/cli
 go 1.21.0
 
 require (
-	dario.cat/mergo v1.0.0
+	dario.cat/mergo v1.0.1
 	github.com/containerd/platforms v0.2.1
 	github.com/creack/pty v1.1.21
 	github.com/distribution/reference v0.6.0
 	github.com/docker/cli-docs-tool v0.8.0
 	github.com/docker/distribution v2.8.3+incompatible
-	github.com/docker/docker v27.2.1+incompatible
+	github.com/docker/docker v27.2.2-0.20240912213415-bf60e5cced83+incompatible // 27.x branch / v27.3.0-dev
```

Comment:
v27.3.0-dev?

Reply:
Yeah, this will be v27.3.0 - go module pseudo versions are always odd (if that's the part you were confused about?)